apps/sms/views/conversation/test/unit/information_test.js

```diff
@@ -427,7 +426,7 @@ suite('Information view', function() {
       });
     });
 
-    test('Outgoing Message report(MMS w/ subject)', function() {
+    test.skip('Outgoing Message report(MMS w/ subject)', function() {
```

Comment:
> I uncommented and it just worked. I don't get it. But I added the readTimestamp to the mock anyway

Weird because when I uncommented it didn't for me :)